root/blog/whats-new-in-go1.27.md

```
@@ -0,0 +1,973 @@
+
+```go
+type Adder interface{ Add(int) int }
+
+type G struct{}
+
+func (G) Add[T any](v T) T { return v }
+
+var _ Adder = G{}
+```
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)